src/Atmos/Model/orientation.jl

```diff
@@ -15,6 +15,7 @@ end
 gravitational_potential(::Orientation, aux::Vars) = aux.orientation.Φ
 ∇gravitational_potential(::Orientation, aux::Vars) = aux.orientation.∇Φ
 altitude(orientation::Orientation, aux::Vars) = gravitational_potential(orientation, aux) / grav
+altitude(aux::Vars) = altitude(aux.orientation, aux)
```

Comment:
It turns out that this won't work because `orientation` is actually a field in `atmos::AtmosModel`, not the one in `aux::Vars`.

Reply:
So what's the right way to write the `altitude()` that Held Suarez wants?

Reply:
You'll need to pass `atmos::AtmosModel` down to where you want to call it, and then call it with `altitude(atmos, aux)`

Reply:
(or pass `orientation` to where you need it, but we may later find that we need other things, so I figure this will be a nice safety net to avoid changing the interface many times)

Reply:
This is needed in `init_state!()` and in a source. We could get the model in the latter by defining an `atmos_source!()`, but not sure how we'd get it in the former. Will likely defer this for this PR.